### ISSUE TITLE
128.npz voxel files are not TDF

### ISSUE THREAD

**Issue by 4ri4Slr** [xiumingzhang/GenRe-ShapeHD#65]
Hi.

I went through the closed issues and it was mentioned that you used TDF for view-centered supervision. However, when I downloaded the dataset, the npz files were in the occupancy grid format. Can you please clarify which method was used? Also, if occupancy grids are the used format, do you have any piece of code available to this end?

Thanks,
Aria

**Comment by xiumingzhang** [comment]
Hi, we used voxels instead of TDF for supervision. Could you point me to the issue that mentioned we used TDF so that I could further clarify on this confusion? Thanks.

**Comment by 4ri4Slr** [comment]
Thanks for getting back to me @xiumingzhang.

It was mentioned on issue #46 that TDFs were used for supervision.  If possible, I would be grateful if you mention the voxelization tool that was used or provide a piece of code for it. 

Bests,
Aria

**Comment by xiumingzhang** [comment]
Oh there, by "TDF" what we really meant is "voxels with values in [0, 1], as opposed to binary voxel values." Sorry for the confusion, and please feel free to reopen if you have more questions.

**Comment by 4ri4Slr** [comment]
I computed view-centered binary voxels using the binvox  and then converted them to npz files. The way I do view-centered supervision is fixing the camera at a constant distance from the origin and then I rotate the object and get the renders for different views. I tried training marrnet 2 with the default settings for 100 epochs and the loss for the last 10s of epochs oscillated around 0.05. I then tried finetuning the marrnet using the pre-trained marrnet1 network, but the reconstruction results are not looking good at all! 
I have attached the predicted output for a sample shape and also provided the voxel and depth maps used for training. 

Thanks for your help.
[samples.zip](https://github.com/xiumingzhang/GenRe-ShapeHD/files/5479420/samples.zip)
